ansible/roles/java-base/tasks/main.yml

```diff
@@ -95,10 +95,20 @@
     - java.rc > 0 and not os|startswith("macos")
   package: name="{{ java_package_name }}" state=present
 
+- name: install java tap (macOS)
+  become_user: administrator
+  when: java.rc > 0 and os|startswith("macos")
+  homebrew_tap:
+        name: AdoptOpenJDK/openjdk
+        state: present
```

Comment:
This is effectivly duplicated code from https://github.com/nodejs/build/blob/1cbb6cf82aaa2646965ff2c15128220ecfe4de16/ansible/roles/package-upgrade/tasks/partials/brew.yml#L36-L39 but I needed this change to get be able install the openjdk, even though ansible tells me the task ive linked to ran ok. @nodejs/build anyone have an ideas/experience as to why this code works in one place but not another?

Reply:
So partials/brew.yml ran and you saw it running homebrew_tap for the AdoptOpenJDK/openjdk cask and the next task, which installs the package, failed until you yanked the homebrew_tap into this position? It's not an ordering problem is it, partials/brew.yml ran before this didn't it?

Reply:
Yeah I can see partials/brew,yml run but it reports nothing changed under `add AdoptOpenJDK repo` it wasnt until I add the tap to this position that anything happened